index.html

```diff
@@ -26,47 +42,70 @@ <h2>Add New Rate</h2>
           </form>
         </div>
         <div class="converter">
-            <h2>Currency Converter</h2>
+          <h2>Currency Converter</h2>
           <form id="converterForm">
             <label for="amount">Amount:</label>
             <input type="number" id="amount" name="amount" required />
 
-            <label for="fromCurrency">From:</label>
-            <select id="fCurrency" name="fromCurrency" required>
-              <option value="USD">USD</option>
+            <label for="convertFrom">From:</label>
+            <select id="convertFrom" name="convertFrom" required>
+              <option value="DKK">DKK</option>
               <option value="EUR">EUR</option>
+              <option value="NGN">NGN</option>
+              <option value="JPY">JPY</option>
+              <option value="CAD">CAD</option>
+              <option value="USD">USD</option>
+              <option value="GHC">GHC</option>
             </select>
 
-            <label for="toCurrency">To:</label>
-            <select id="tCurrency" name="toCurrency" required>
-              <option value="USD">USD</option>
+            <label for="convertTo">To:</label>
+            <select id="convertTo" name="convertTo" required>
+              <option value="DKK">DKK</option>
               <option value="EUR">EUR</option>
+              <option value="NGN">NGN</option>
+              <option value="JPY">JPY</option>
+              <option value="CAD">CAD</option>
+              <option value="USD">USD</option>
+              <option value="GHC">GHC</option>
             </select>
 
             <button type="button" onclick="convertCurrency()">Convert</button>
           </form>
+          <p id="result"></p>
         </div>
 
         <div class="converter">
-            <h2>Update Currency</h2>
+          <h2>Update Currency</h2>
           <form id="updateRateForm">
-            <label for="fromCurrency">From Currency:</label>
-            <input type="text" id="ufromCurrency" name="fromCurrency" required />
+            <label for="baseCurrencyUpdate">From Currency:</label>
+            <input
```

Comment:
You used a dropdown for the other forms. I think this one could benefit from having a dropdown with the currencies currently having a rate.